frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -217,9 +225,12 @@ export default class Metadata extends React.Component<Props> {
             { this.renderSources(entity) }
             <Localized id='entitydetails-metadata-resource' attrs={ { title: true } }>
                 <Property title='Resource' className='resource'>
-                    <Link to={ `/${locale.code}/${entity.project.slug}/${entity.path}/` }>
+                    <a
```

Comment:
Why replace `Link` with `a`? If it's to use `onClick`, I'd be surprised if it didn't work with `Link`.

Reply:
What's the benefit of using `Link` instead of `a`? I'm preventing default anyways (to check for unsaved changes).

Reply:
To me, it helps understanding what the link is about. I'm using `<a>` when I link to an external resource (a different page, a different website) and `<Link>` when I link to an internal resource, meaning when it doesn't trigger a page reload. It doesn't make much difference but I think it makes the intent clearer.

Reply:
Good point about semantics. I'll update.

Reply:
Meh, the problem with `<Link>` is that it prepends `/translate` to the `href` attribute, and it doesn't allow me to navigate to the right URL without manually stripping `/translate`.